examples/amp-story/amp-story-audio-sticker.html

```diff
@@ -207,7 +207,7 @@ <h1 class="bold">Key Highlights of AMP Conf 2018</h1>
           <p class="byline">By The AMP team</p>
         </amp-story-grid-layer>
         <amp-story-grid-layer>
-          <amp-story-audio-sticker>
+          <amp-story-audio-sticker style="outline" outline-color="rgb(255, 0, 0)">
```

Comment:
Would love to know more about this attribute shape. The attribute `style` is generally reserved for CSS styling, and in this case the HTML will try to parse it as a list of CSS rules.

Can we just use `outline-color` (without style=outline), or just rely on creators using `style="border: 1px solid red"`? Can you also add screenshots of the desired effect on the PR description? A demo URL might be even better to try this. From what I see here, this API doesn't offer anything that creators could not do with their own `style="box-shadow: abc"` declaration.

Also, it might be best to implement the border as a box-shadow anyways so that the position doesn't change between the two styles (borders increase the size of the element, box shadows don't).

Reply:
Also, I see that the second part of this PR is the expand/shrink functionality. Do you want to focus on that part instead of this for this PR?

Reply:
Thanks for the catch! Using `sticker-style` instead. Yes you are right this is merely just a default style that developers can directly use without any code. You can check more detailed UX requirements in the reference above in the description.

Done. Good idea!